sdk/core/Test-Setup.ps1

```diff
@@ -57,9 +57,11 @@ if ($PackageName -eq "azure_core_amqp") {
     # now that the Test broker has been built, launch the broker on a local address.
     $env:TEST_BROKER_ADDRESS = 'amqp://127.0.0.1:25672'
 
-    $brokerExecutable = $WorkingDirectory'/azure-amqp/bin/Debug/TestAmqpBroker/net462/TestAmqpBroker.exe'
-    $remoteJob = Start-Process $WorkingDirectory/azure-amqp/bin/Debug/TestAmqpBroker/net462/TestAmqpBroker.exe -ArgumentList { $env:TEST_BROKER_ADDRESS, "/headless" }
+    $brokerExecutable = $WorkingDirectory + '/azure-amqp/bin/Debug/TestAmqpBroker/net462/TestAmqpBroker.exe'
+    Write-Host "Test broker is: $brokerExecutable"
     Write-Host "Starting test broker listening on " $env:TEST_BROKER_ADDRESS "..."
+
+    Start-Process $brokerExecutable -ArgumentList { $env:TEST_BROKER_ADDRESS, "/headless" }
```

Comment:
Also, you're passing a scriptblock to `-ArgumentList` when it should be just an array. Either:

```suggestion
    Start-Process $brokerExecutable -ArgumentList $env:TEST_BROKER_ADDRESS, "/headless"
```

Or make it explicit, though this is really only used to force possibly-single items to arrays:

```suggestion
    Start-Process $brokerExecutable -ArgumentList @( $env:TEST_BROKER_ADDRESS, "/headless" )
```